### PR TITLE
Update NEWS

### PR DESCRIPTION
**Pull request by iglesias**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -48,6 +48,12 @@
 		- Removing redundant and fixing implicit imports [Thoralf Klein]
 		- Hide many methods from SWIG, reducing compile memory by 500MiB [Heiko Strathmann, Fernando Iglesias, Thoralf Klein]
 
+2014-12-16 Fernando Iglesias <fernando.iglesias@shogun-toolbox.org>
```
Comment (karlnapf) [inline NEWS]: Could you also add the "new" news section? This is the one of the old release...

Comment (karlnapf) [inline NEWS]: Wait I am confused. Why did you add this here? Looking at the file doesnt seem it has the correct structure now

Comment (iglesias) [inline NEWS]: That seems to be true. I had this old commit in a local branch and pushed it overlooking the new structure.

Comment (iglesias) [inline NEWS]: What is the "new" news section?

Comment (iglesias) [inline NEWS]: I will move this into Cleanups in the section above. I am wondering though, how do keep track of news constantly if we put everything in a section after a release?
